File: question14.py
# klavyeden girilen 25 adet sayidan negatif olanlarin toplamini, cift olanlarin carpimini ve 7'ye esit olanlarin adedini bulup ekrana yazdiran kod.

negTop=0
ciftCarp=1
yedi=0

for i in range(0,25):
    sayi=(int(input("Bir sayi giriniz:")))
    if(sayi<0):
        negTop=negTop+sayi
    if(sayi%2==0):
        ciftCarp=ciftCarp*sayi
    if(sayi==7):
        yedi=yedi+1

print("Negatif sayilarin toplami:" + str(negTop))

print("Cift sayilarin carpimi:" + str(ciftCarp))

print("Yedi adedi: " + str(yedi))

File: question14.java
import java.util.Scanner;
public class question14 {
    public static void main(String[] args) {
        // Klavyeden girilen 25 adet sayi icerisinde negatif olanlarin toplamini, cift sayilarin carpimini, 7'ye esit olanlarin adedini bulup ekrana yazdiran kod.

        Scanner scanner = new Scanner(System.in);

        int negTop=0;
        int ciftCarp=1;
        int yedi=0;

        for(int i=0;i<25;i++){
            System.out.print("Bir sayi giriniz: ");
            int sayi=scanner.nextInt();
            if(sayi<0){
                negTop+=sayi;
            }
            else if(sayi%2==0){
                ciftCarp*=sayi;
            }
            else if(sayi==7){
                yedi++;
            }
        }

        System.out.print("Negatif sayilarin toplami: " + negTop);
        System.out.print("Cift sayilarin carpimi:" + ciftCarp);
        System.out.print("7 sayisinin adedi: " + yedi);

        scanner.close();
    }
}
